### PR TITLE
ZEPPELIN-383 Override classpath with ZEPPELIN_CLASSPATH_OVERRIDES

### PR DESCRIPTION
**Pull request by echarles**
Prepend ZEPPELIN_CLASSPATH_OVERRIDES environment variable when building CLASSPATH in the shell scripts.

This PR replaces the closed  #398 and #386 - Sorry for the mess...

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
merge if there're no more discussions

**Comment by felixcheung** [comment]
looks good

File: bin/interpreter.sh [hunk 1]
```diff
@@ -121,9 +121,9 @@ fi
 CLASSPATH+=":${ZEPPELIN_CLASSPATH}"
 
 if [[ -n "${SPARK_SUBMIT}" ]]; then
-    ${SPARK_SUBMIT} --class ${ZEPPELIN_SERVER} --driver-class-path "${CLASSPATH}" --driver-java-options "${JAVA_INTP_OPTS}" ${SPARK_SUBMIT_OPTIONS} ${SPARK_APP_JAR} ${PORT} &
+    ${SPARK_SUBMIT} --class ${ZEPPELIN_SERVER} --driver-class-path "${ZEPPELIN_CLASSPATH_OVERRIDESS}:${CLASSPATH}" --driver-java-options "${JAVA_INTP_OPTS}" ${SPARK_SUBMIT_OPTIONS} ${SPARK_APP_JAR} ${PORT} &
```
Comment (minahlee) [inline bin/interpreter.sh]: @echarles Variable name ends with double S. Maybe its typo?

File: bin/interpreter.sh [hunk 2]
```diff
@@ -121,9 +121,9 @@ fi
...
 else
-    ${ZEPPELIN_RUNNER} ${JAVA_INTP_OPTS} -cp ${CLASSPATH} ${ZEPPELIN_SERVER} ${PORT} &
+    ${ZEPPELIN_RUNNER} ${JAVA_INTP_OPTS} -cp ${ZEPPELIN_CLASSPATH_OVERRIDESS}:${CLASSPATH} ${ZEPPELIN_SERVER} ${PORT} &
```
Comment (minahlee) [inline bin/interpreter.sh]: same here

**Pull request closed without merging**